ert_shared/models/base_run_model.py

```diff
@@ -214,12 +214,10 @@ def is_forward_model_finished(progress):
         return not (any((job.status != "Success" for job in progress)))
 
     def update_progress_for_index(self, iteration, idx, run_arg):
-        if not self._run_context.is_active(idx):
-            return
         try:
             # will throw if not yet submitted (is in a limbo state)
             queue_index = run_arg.getQueueIndex()
-        except ValueError:
+        except (ValueError, AttributeError):
```

Comment:
Removing the above check will raise `AttributeError` because `run_arg` might not be set, but removing the check is necessary in order to get updates for failing realisations (as they are deactivated in libres immediately after failure).